react-native/ios/RealmReact/RealmReact.mm

```diff
@@ -317,7 +320,19 @@ - (void)setBridge:(RCTBridge *)bridge {
             }
 
             _initializeOnJSThread(^{
-                return [bridge jsContextRef];
+                // RN < 0.58 has a private method that returns the js context
+                if ([bridge respondsToSelector:@selector(jsContextRef)]) {
+                    return [bridge jsContextRef];
+                }
+                // RN 0.58+ wraps the js context in the jsi abstraction layer,
+                // which doesn't have any way to obtain the JSGlobalContextRef,
```

Comment:
Should we create an issue on ReactNative about this? Since the JSI abstraction is their new architecture going forward. So having public API's for it would be nice.

Reply:
The JSI interface isn't public either and when we asked for a way to get the context in the past they said no.